components/atoms/circleImage.html

```diff
@@ -0,0 +1,11 @@
+<div class="component" title="Circle Image" version="0.1">
```

Comment:
first the version was 1.0. after we make changes, we need to increase the version number by 0.1. so is should be 1.1
<div class="component" title="Circle Image" version="1.1">